_includes/keynotes.html

```diff
@@ -6,9 +6,10 @@ <h2>Keynote Speakers</h2>
 			<!--<a style="background-image: url(/assets/images/keynotes/{{speaker.image | default: 'owasp_logo.png'}});{{speaker.style}};" alt="{{speaker.name}}"
 			href="/program/keynotes#{{speaker.name}}"
 			></a>-->
-		        <a style="background-image: url(/assets/images/keynotes/{{speaker.image | default: 'owasp_logo.png'}});{{speaker.style}};" alt="{{speaker.name}}"
```

Comment:
the alt-attribute should be kept to aid visually impared users, but it's fine to add the name outside as well.

Reply:
Let's keep both the alt and the name under the keynote